vpn-router-core/src/main/java/vpnrouter/core/infrastructure/shell/RealTunnelling.java

```diff
@@ -0,0 +1,46 @@
+package vpnrouter.core.infrastructure.shell;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import vpnrouter.core.service.client.client.Tunnelling;
+
+import java.util.Set;
+
+import static java.util.stream.Collectors.joining;
+
+@Primary
+@Component
+@Profile("prod")
+@RequiredArgsConstructor
+public class RealTunnelling implements Tunnelling {
+
+    private static final int TABLE = 7373;
```

Comment:
why is this value set for TABLE?

Reply:
https://habr.com/ru/articles/108690/
http://linux-ip.net/html/routing-tables.html
https://man7.org/linux/man-pages/man8/ip-rule.8.html

There are 4 default routing tables: unspec, local, main, and default. They have IDs 0, 255, 254, and 253 correspondingly. The default tables are declared in `/etc/iproute2/rt_tables`.

Therefore, we can choose any number (idk the range) except those default IDs for our custom routing table.

Reply:
Thanks) After reading this links, I got some insights about it, but I'd like to discuss it one more time when we have a call)